recipe/meta.yaml

```diff
@@ -34,33 +35,35 @@ requirements:
     - pip
     - cython >=0.29.24,!=0.29.18
     - meson-python
-    - numpy
+    - numpy 1.22                             # [py<311]
+    - numpy 1.23                             # [py==311]
     - pythran
     - packaging >=20.0
   run:
     - python
-    - numpy
+    - {{ pin_compatible('numpy') }}
     - scipy >=1.8,<1.9.2  # [py <= 39]
```

Comment:
why is this scipy pin important?

Reply:
it comes from #102, but the discussion there suggests that indeed it is no longer needed. 🙏